Exercise_merge_conflict/Exercise_6.ipynb

```diff
@@ -0,0 +1,512 @@
+   "metadata": {},
+   "source": [
+    "### Add a remote repository\n",
+    "\n",
+    "Let's add the remote repository to our local repository like in the previous exercise.\n",
+    "\n",
+    "The setup script has already created one that you can use at: \n",
+    "\n",
+    "**../conference_planning_remote**\n",
+    "\n",
+    "Use the line above as *remote_path* to the remote repository."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "discrete-affiliation",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# use \"git remote add <some_remote_name> <remote_path>\" to add the remote \n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "complicated-fifth",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get information from remote branch with \"git fetch <my_remote>\n",
+    "\n",
+    "# Check which branches are available\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bb0bba96",
+   "metadata": {},
+   "source": [
+    "### Now make a different change in the same file on the updated_schedules branch\n",
+    "  * Switch to the updated_schedules branch\n",
+    "  * Change the workshop to a talk\n",
+    "  * Commit your changes"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9f3a660d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Switch to updated_schedules\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "limiting-snowboard",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Change workshop to talk and commit\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ef95def7",
+   "metadata": {},
+   "source": [
+    "### Merge the updated_schedules branch into the main branch\n",
+    "We decided to use the schedule on the updated_schedules branch. Therfore, we want to merge this branch into the main branch."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8832fdb5",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Go to main branch and merge the updated_schedules branch\n",
+    "# make sure to always add a commit message when merging with -m <my commit message>\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "automatic-underground",
+   "metadata": {},
+   "source": [
+    "### Solve merge conflict\n",
+    "If you've done everything \"right\", something has gone \"wrong\" and the output should look like the following:\n",
+    "```\n",
+    "CONFLICT (content): Merge conflict in schedule_day1\n",
+    "Automatic merge failed; fix conflicts and then commit the result.\n",
+    "```\n",
+    "\n",
+    "We apparently run into a merge conflict. This happened because we did changes on the main branch and the updated_schedules branch at the same part of the file and Git doesn't know which changes it should take. Let's see how to solve this. \n",
```

Comment:
In this part users need a hint to do a git commit -m after the unsucessful merge. Otherwise the editor will open.
And somewhere we need to tell the users that the -m for the merge is just for safety reasons, not good practice